QPM/SymLinker/LinkCreators/WindowsSymLinkCreator.cs

```diff
@@ -0,0 +1,30 @@
+﻿using SymLinker;
+using System;
+using System.IO;
+using System.Runtime.InteropServices;
+
+namespace SymLinker.LinkCreators
+{
+    internal class WindowsSymLinkCreator : ISymLinkCreator
+    {
+        [DllImport("Kernel32.dll", CharSet = CharSet.Unicode )]
+        static extern bool CreateHardLink(
+            string lpFileName,
+            string lpExistingFileName,
+            IntPtr lpSecurityAttributes
+        );
+
+        public bool CreateSymLink(string source, string dest, bool file)
+        {
+            var symbolicLinkType = file ? SymbolicLink.File : SymbolicLink.Directory;
+            return CreateHardLink(dest, source, (IntPtr) symbolicLinkType);
+        }
+
```

Comment:
Could also add stuff here for junctions, if you want.

Reply:
I cannot as of the current dependency system. I _can_ work on a cache overhaul to version the fetched dependencies so we can symlink header folders and `overrideSoName` libs too, however I think it's outside of the scope of the PR